nightfall-optimist/src/routes/proposer.mjs

```diff
@@ -503,11 +505,25 @@ router.get('/stake', auth, async (req, res, next) => {
  *          $ref: '#/components/responses/InternalServerError'
  */
 router.get('/withdraw', auth, async (req, res, next) => {
+  const ethAddress = req.app.get('ethAddress');
+  const ethPrivateKey = req.app.get('ethPrivateKey');
+
   try {
-    const proposersContractInstance = await getContractInstance(PROPOSERS_CONTRACT_NAME);
-    const txDataToSign = await proposersContractInstance.methods.withdraw().encodeABI();
+    // Recreate State contract
+    const stateContractInstance = await getContractInstance(STATE_CONTRACT_NAME);
 
-    res.json({ txDataToSign });
+    const txDataToSign = await stateContractInstance.methods.withdraw().encodeABI();
+    const tx = {
+      from: ethAddress,
+      to: stateContractInstance.options.address,
+      data: txDataToSign,
+      gas: 8000000,
+    };
+    const signedTx = await web3.eth.accounts.signTransaction(tx, ethPrivateKey);
+    const receipt = await web3.eth.sendSignedTransaction(signedTx.rawTransaction);
+    logger.debug(`Transaction receipt ${receipt}`);
```

Comment:
Every object returned by the endpoints are automatically logged, then I think these `logger.debug` instructions that logs these objects can be removed.